### DISCUSSION TITLE
Picker - reset dropdown state

### DISCUSSION THREAD

**Discussion by chrisphillers** [adobe/react-spectrum#2378] [Q&A] [answered]
Hi,
I want to reset my form but an unable to reset the Picker to default "null" state. 

I can reset TextArea/TextFields since the value is controlled, but the Picker is not. I cannot find any info in the documentation/props. Is anyone aware of a way to do this (apart from reloading the page, its a bit too ungraceful).

Thanks
Chris

**Comment by snowystinger** [comment]
Picker is like a radio, you can't unselect something. Instead, you have to offer none as an option.

**Comment by chrisphillers** [comment, marked as answer]
I have actually come up with the solution, you can control what is selected using "selectedKey" so you can "unselect" all options if you set it to null.
